crates/bevy_pbr/src/cluster/mod.rs

```diff
@@ -63,6 +63,27 @@ const CLUSTER_COUNT_MASK: u32 = (1 << CLUSTER_COUNT_SIZE) - 1;
 // The z-slicing method mentioned in the aortiz article is originally from Tiago Sousa's Siggraph 2016 talk about Doom 2016:
 // http://advances.realtimerendering.com/s2016/Siggraph2016_idTech6.pdf
 
+#[derive(Resource)]
+pub struct GlobalClusterSettings {
+    pub supports_storage_buffers: bool,
+    pub clustered_decals_are_usable: bool,
+}
+
+pub(crate) fn make_global_cluster_settings(world: &World) -> GlobalClusterSettings {
```

Comment:
Note: this is intentionally not impl FromWorld: GlobalClusterSettings will end up in bevy_camera crate where bevy_render is not available, and this function needs bevy_render. the orphan rule says you cannot impl a foreign trait on a foreign type.